pkgs/by-name/fi/firefly-iii-data-importer/package.nix

```diff
@@ -81,6 +81,8 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     ln -s ${dataDir}/cache $out/bootstrap/cache
   '';
 
+  env.COMPOSER_MIRROR_PATH_REPOS = "1";
```

Comment:
I would definitely update the `buildComposerProject` builder so that this is set by default instead.

Reply:
Thing is that it is set by default in `buildComposerProject`. But if we don't use it it has to be set by us.

Reply:
I believe this is because you're using `mkComposerRepository` instead of `buildComposerProject`. I still continue to think that it should be fixed in there.

Reply:
That said, it is set in `mkComposerRepository` ! https://github.com/NixOS/nixpkgs/blob/eaed87267b5f6e2700f84d86097bfcc95b533e2b/pkgs/build-support/php/builders/v1/build-composer-repository.nix#L107

Reply:
I don't think it matters in `mkComposerRepository` because that is not building the `vendor` directory in the final package. It needs to be set in the `composerInstallInstall` hook. However, that would require exposing it as an additional variable for the hook.

Reply:
Working on it.